.evg.yml

```diff
@@ -152,71 +134,69 @@ tasks:
       - func: "setup_mongod"
       - func: "setup_stitch"
       - func: "setup_stitch_swift"
-      - func: "setup_test_creds"
       - command: shell.exec
         params:
           shell: "bash"
           script: |
-            echo "running ios tests"
+            echo "running ios tests"           
             set -e
 
             if pgrep CoreSimulator; then pkill -9 CoreSimulator; fi
+
             export DEVELOPER_DIR=/Applications/Xcode9.2.app
             cd stitch-ios-sdk
             export SIM_UUID=$(xcrun simctl create stitch-iphone7 com.apple.CoreSimulator.SimDeviceType.iPhone-7 com.apple.CoreSimulator.SimRuntime.iOS-10-2)
             xcrun simctl boot $SIM_UUID
             echo "booting SIMCTL $SIM_UUID"
-            
-            cc=`find ./.build/checkouts/ -type d -name "CommonCrypto*" -print`
-
-            run_xctest() {
-              local xcodeproj=`find . -type d -name "$1.xcodeproj" -print`
-              local scheme=$2
-              local no_test=$3
-              echo "...testing $1"
-
-              xcodebuild \
-                -project "`pwd`/$xcodeproj" \
-                -destination "id=$SIM_UUID" \
-                -derivedDataPath "localDerivedData" \
-                -scheme $scheme \
-                OTHER_LDFLAGS="-rpath `pwd`/vendor/MobileSDKs/iphoneos/lib" \
-                LIBRARY_SEARCH_PATHS="`pwd`/vendor/MobileSDKs/iphoneos/lib" \
-                SWIFT_INCLUDE_PATHS="`pwd`/$cc `pwd`/vendor/MobileSDKs/include `pwd`/vendor/MobileSDKs/include/libbson-1.0 `pwd`/vendor/MobileSDKs/include/libmongoc-1.0 `pwd`/vendor/MobileSDKs/include/mongo/embedded-v1/" \
-                FRAMEWORK_INCLUDE_PATHS="`pwd`/localDerivedData" \
-                ENABLE_BITCODE=NO \
-                IPHONEOS_DEPLOYMENT_TARGET="8.0" \
-                `[[ $no_test != YES ]] && echo "test" || echo ""`
-            }
 
             echo "...testing core"
-            run_xctest StitchCoreSDK StitchCoreSDK-Package
-            run_xctest StitchCoreAdminClient StitchCoreAdminClient-Package
-            run_xctest StitchCoreTestUtils StitchCoreTestUtils-Package
+            ./scripts/run_xctest.sh StitchCoreSDK StitchCoreSDK-Package NO $SIM_UUID
```

Comment:
[opt] given the repetitive nature of this, you could create a bash array containing each module of each section to run the tests on. Would cut down the script size a bit.